enterprise/views.py

```diff
@@ -30,9 +30,11 @@
     lift_quarantine = None
 
 # isort:imports-firstparty
-from enterprise.models import EnterpriseCustomer, EnterpriseCustomerUser, UserDataSharingConsentAudit
+from enterprise.lms_api import CourseApiClient
+from enterprise.models import (EnterpriseCourseEnrollment, EnterpriseCustomer, EnterpriseCustomerUser,
+                               UserDataSharingConsentAudit)
 from enterprise.tpa_pipeline import active_provider_enforces_data_sharing, get_enterprise_customer_for_request
-from enterprise.utils import NotConnectedToEdX
+from enterprise.utils import NotConnectedToEdX, consent_necessary_for_course
 
 
 def verify_edx_resources():
```

Comment:
The doc strings in lines 53, 86-87, and 91-92 are the same as some of the doc strings in enterprise/conf/locale/en/LC_MESSAGES/django.po. I'll make the same comments here:

**l. 53**
OpenEdX --> Open edX

**ll. 86-87**
OK

**ll. 91-92**
get --> receive, strike "any"